src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -67,7 +67,8 @@ private AssertionMessages() {
           + ".wds-global-navigation__user-logged-in .wds-avatar img";
   
   private static final String WDS_FOOTER_HEADER_CLASS = "wds-global-footer__header";
-  private static final By MERCURY_SKIN = By.cssSelector(".mobile-wiki");
+//  private static final By MERCURY_SKIN = By.cssSelector("#ember-container, .mobile-wiki");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)